backend/repository/config.go

```diff
@@ -78,10 +78,15 @@ type EndptMidConfig struct {
 
 // ClientConfig stores configuration for an client.
 type ClientConfig struct {
-	ID             string
-	Type           ProtocolType
-	ThriftFile     string
-	ExposedMethods map[string]string
+	Name              string            `json:"name"`
+	Type              ProtocolType      `json:"type"`
+	ThriftFile        string            `json:"thriftFile,omitempty"`
+	MuttleyName       string            `json:"muttleyName,omitempty"`
```

Comment:
`ServiceName` is overload for http. what could be a better one? `TchannelServiceName`, `TchannelRoutingName`?